src/index.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import Router from './Components/Router';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <Router />
```

Comment:
It would be better to render <App> here instead of Router and move Router to the App.js file, because usually App component is the main component that is always rendered. 

You can move the content of the homepage to a separate Home.js component and keep there
```
<h1> Search Bar </h1>
      <SearchBar />
```

In App.js file you can store Router. :)